webviz_ert/controllers/response_correlation_controller.py

```diff
@@ -551,6 +568,16 @@ def _get_selected_ensembles_from_store(
             ]
         return None
 
+    def _format_index_value(axis: pd.Index, index: int) -> Union[int, datetime.date]:
+        raw_value = axis[index]
+        if isinstance(raw_value, str):
+            try:
+                datetime_value = pd.to_datetime(raw_value)
+                return datetime_value.date()
+            except (pd.errors.ParserError):
+                pass
+        return raw_value
+
 
 def sort_dataframe(
```

Comment:
Should it be internal `_`?

Reply:
the `sort_dataframe` function? outside the scope of this PR, but following the campsite rule, i guess we can make it internal. i tried to make it reusable, but the part where you check if the index exists, that's probably very particular to our case.

Reply:
done